src/main/kotlin/com/manimdsl/runtime/ASTExecutor.kt

```diff
@@ -79,19 +81,30 @@ class VirtualMachine(
         return if (result is RuntimeError) {
             addRuntimeError(result.value, result.lineNumber)
             Pair(ExitStatus.RUNTIME_ERROR, linearRepresentation)
-        } else {
+        } else if (!stylesheet.userDefinedPositions()) {
             val (exitStatus, computedBoundaries) = Scene().compute(dataStructureBoundaries.toList(), hideCode)
+            if (returnBoundaries) {
+                val gson = Gson()
+                println(gson.toJson(computedBoundaries.map { it.key to it.value.positioning() }.toMap()))
```

Comment:
What is being printed here? If it's extendable, could it be moved into a function (I'm thinking of something like `RuntimeError` which basically wraps a print statement)

Reply:
It's the boundaries for our web ui, we need to get out the boundaries per ds to show the user before they move stuff around, so that's why. Don't think it's really extensible